#*H
#  Copyright 2017 Transport Systems Catapult (The Catapult)
#  All rights reserved
#  For use only for the purpose agreed with The Catapult
#H*

Header header
PodHeader pod_header
uint8 seed
uint8 key_status
uint8 session_status	# Also session restart request?
uint32 key_response
uint8 qos
